src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/config/remote_vetting.yml

```diff
@@ -5,7 +5,10 @@ services:
 
   Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\ServiceProviderFactory:
     arguments:
-      - '%remote_vetting_sp%'
+      $router: '@router'
+      $entityId: "%remote_vetting_entity_id%"
+      $assertionConsumerUrlSlug: "ss_second_factor_remote_vet_acs"
```

Comment:
Why hard-code it in the service container? Might be more predictable to hard code it in code?

Reply:
The slug of the route itself isn't hardcoded either so the slug is dependency of the ServiceProviderFactory and that's why i used DI.